src/test/java/com/example/todowithcouchbase/task/service/impl/TaskServiceImplTest.java

```diff
@@ -0,0 +1,100 @@
+package com.example.todowithcouchbase.task.service.impl;
+
+import com.example.todowithcouchbase.task.exception.TaskWithThisNameAlreadyExistException;
+import com.example.todowithcouchbase.task.model.dto.request.SaveTaskRequest;
+import com.example.todowithcouchbase.task.model.dto.response.SaveTaskResponse;
+import com.example.todowithcouchbase.task.model.entity.TaskEntity;
+import com.example.todowithcouchbase.task.model.mapper.SaveTaskRequestToTaskEntityMapper;
+import com.example.todowithcouchbase.task.model.mapper.TaskEntityToTaskResponseMapper;
+import com.example.todowithcouchbase.task.repository.TaskRepository;
+import com.example.todowithcouchbase.base.AbstractBaseServiceTest;
+import com.example.todowithcouchbase.builder.SaveTaskRequestBuilder;
+import com.example.todowithcouchbase.builder.TaskEntityBuilder;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+
+class TaskServiceImplTest extends AbstractBaseServiceTest {
+
+    @InjectMocks
+    private TaskServiceImpl taskService;
+
+    @Mock
+    private TaskRepository taskRepository;
+
+
+    private final SaveTaskRequestToTaskEntityMapper saveTaskRequestToTaskEntityMapper =
+            SaveTaskRequestToTaskEntityMapper.initialize();
+
+    private final TaskEntityToTaskResponseMapper taskEntityToTaskResponseMapper=
```

Comment:
- Use TaskEntityToTaskMapper here.

Reply:
we do not need to use it in test class  instead of using it i deleted it

Reply:
@artfulCoder98 you couldn't fix it.